google-cloud-errors/lib/google/cloud/errors.rb

```diff
@@ -231,6 +231,13 @@ def self.gapi_error_class_for http_status_code
     ##
     # Canceled indicates the operation was cancelled (typically by the caller).
     class CanceledError < Error
+      ##
+      # gRPC error code for CANCELLED
+      #
+      # @return [Integer]
+      def code
+        1
```

Comment:
Should we define constants like Error::CanceledError = 1 and then use the constant here as return value? This way all constants can be declared in one place. It can help with tests too, instead of checking individual values